components/Panel/VotePanel/Details.tsx

```diff
@@ -69,6 +68,13 @@ export function Details({
     setShowRawClaimData(!showRawClaimData);
   }
 
+  const augmentedDataResponse = useAugmentedVoteData({
+    time,
+    identifier: decodedIdentifier,
+    ancillaryData: ancillaryData,
+  });
+  const augmentedData = augmentedDataResponse.data;
+
```

Comment:
this used to be fetched in the vote context for all votes. now we move this to the render component, and only fetch a single augmented data per vote when we are viewing the details pane.  much more efficient, and easier to cache